src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -874,21 +781,49 @@ void Graph::Allocate() {
     for (auto& edge : graphEdges) edge->validate();
 }
 
-void Graph::PushInputData(const std::string& name, const InferenceEngine::Blob::Ptr &in) {
+void Graph::PushInputData(const std::string& name, const ov::Tensor &in) {
     if (!IsReady()) IE_THROW()<< "Wrong state. Topology not ready.";
 
     auto input = inputNodesMap.find(name);
     if (input != inputNodesMap.end()) {
-        auto& inTensorDesc = in->getTensorDesc();
+        auto create_tensor_desc = [&](const ov::Tensor& tensor) -> InferenceEngine::TensorDesc {
+            auto element_type = tensor.get_element_type();
+            auto shape = tensor.get_shape();
+            std::vector<size_t> blk_order(shape.size());
+            std::iota(blk_order.begin(), blk_order.end(), 0);
+            std::vector<size_t> dim_offset(shape.size(), 0);
+            std::vector<size_t> blk_strides;
+            auto byte_strides = element_type.bitwidth() >= 8 ? tensor.get_strides() : Strides{};
+            if (byte_strides.empty()) {
+                blk_strides = ov::row_major_strides(shape);
+            } else {
+                blk_strides.resize(byte_strides.size());
+                std::transform(byte_strides.begin(),
+                               byte_strides.end(),
+                               blk_strides.begin(),
+                               [&element_type](size_t byte_stride) {
+                                   OPENVINO_ASSERT(byte_stride % element_type.size() == 0,
+                                                   "Limitation: Stride in bytes ",
+                                                   byte_stride,
+                                                   " should be divisible by size of element ",
+                                                   element_type.size());
+                                   return byte_stride / element_type.size();
+                               });
+            }
+            return ie::TensorDesc{ie::details::convertPrecision(element_type),
+                                  shape,
+                                  ie::BlockingDesc{shape, blk_order, 0, dim_offset, blk_strides}};
+        };
+        auto inTensorDesc = create_tensor_desc(in);
```

Comment:
It is a bit awkward to create a fake ie::TensorDesc and then map to cpu_plugin MemoryDesc. Maybe we can directly create the memory object from precision and shape. @maxnick How do you think?

Reply:
yes, you are right, we need do it, but this PR focus on the interface API change to minimize internal modification as possible. So this PR doesn't change too much logic related the CPU memory object.
We will do CPU memory object related in the following PR, which will do modification only in CPU plugin without plugin api 2.0 interface level.

Reply:
yes, so that we can tensor-->CpuBlockedMemoryDesc to replace tensor --> ie::TensorDesc -->CpuBlockedMemoryDesc